Podcast/EmptyStateView.swift

```diff
@@ -18,6 +18,7 @@ enum EmptyStateType {
     case followers
     case subscription
     case unimplemented
+    case downloads
```

Comment:
Ask Eileen for a larger transparent icon or just make the image nil because this looks super weird.
<img width="145" alt="screen shot 2018-03-01 at 12 00 38 am" src="https://user-images.githubusercontent.com/14966713/36827791-af261b5c-1ce3-11e8-98ac-9cc14c94cf27.png">

Reply:
I did, still waiting on it